src/daily-auths-report.tsx

```diff
@@ -153,6 +153,9 @@ function tabulateSumByAgency(results?: ProcessedResult[], filterIal?: number): T
   };
 }
 
+const formatSIPrefix = format(".2s");
+const formatSIDropTrailingZeroes = (d: number): string => formatSIPrefix(d).replace(/\.0+/, "");
```

Comment:
Is it intended to drop the decimal numbers if there's a non-zero after the zero, e.g. `1000.09` ? 

If not:

```suggestion
const formatSIDropTrailingZeroes = (d: number): string => formatSIPrefix(d).replace(/\.0+$/, "");
```

Reply:
Or are there any digits ever after the `.0` ? 🤔  If yes, then this could cause some strangeness, e.g.

```
> '1000.09'.replace(/\.0+/, '')
'10009'
```

I'd expect one of:

```suggestion
const formatSIDropTrailingZeroes = (d: number): string => formatSIPrefix(d).replace(/\.0/, "");
```

```suggestion
const formatSIDropTrailingZeroes = (d: number): string => formatSIPrefix(d).replace(/\.0.+$/, "");
```

```suggestion
const formatSIDropTrailingZeroes = (d: number): string => formatSIPrefix(d).replace(/\.0+$/, "");
```

Reply:
The `.2` to d3-format says 2 significant digits, so `1000.09` will convert to `1.0k` always.

For safety, I'll add the anchored regex

Reply:
The anchor got messed up by the SI suffix (the "k" character in tests) so I'm going to remove this... precision + tests should cover it for now